### ISSUE TITLE
Judging code including diacritics

### ISSUE THREAD

**Issue by FherStk** [DMOJ/online-judge#2054]
Dear DMOJ team,

I'm loving DMOJ, but I'm unable to perform test cases using code with diacritics like accents. Is there any easy way to allow this? I teach in Catalan language and some of my statements requests some outputs that include some diacritics. 

Here's an example:
1. Source code:
![image](https://user-images.githubusercontent.com/324326/207009445-1f538141-76a1-4344-9fe0-b0fd85da3a3e.png)

2. The judge answer:
![image](https://user-images.githubusercontent.com/324326/207009592-358021bc-2cf8-40f2-a6b1-024cef3c5865.png)

Thanks in advance and best regards,
Fher

**Comment by Xyene** [comment]
By default we use an ASCII-only output stream to speed up Java grading, you can specify `binary_data: true` in your `init.yml` to bypass this.

**Comment by FherStk** [comment]
Great, thanks a lot.

I'm a newbie in DMOJ, I know how to create an init.yml file and store it using a terminal but, is there some way to do it thought the web interface? Can I just add it into the ZIP file with the test cases?

Thanks again and sorry for the inconveniences,
Fher

**Comment by Xyene** [comment]
There is, see the bottom of https://docs.dmoj.ca/#/site/managing_problems

The binary data flag isn't exposed in the site interface, though.

**Comment by FherStk** [comment]
Yes, I used this interface to upload the test cases, my question was about the "init.yml" file. 

I'm migrating my HackerRank problems to DMOJ, and I've ready all the needed scripts to automate these tasks. No problem on this because the files are copied directly to the proper folders. The problem becomes when creating a problem using the web interface.

I performed some tests and I checked that the "init.yml" file is created automatically, but there is no way to set the binary data flag (as you said). I will try to do some changes in the code in order to enable it.

Thanks again!
Fher

**Comment by FherStk** [comment]
Hi again,

I just edited the code and added the field, i tested it and the "binary_data" is being stored correctly into the "init.yml" file. I would be honored if I could contribute, so I created a pull request: #2055

Thanks again for all your support,
Fher

**Comment by kiritofeng** [comment]
As a word of warning, `binary_data: true` will also stop normalizing `\r` and `\r\n` to `\n`; this will work fine with every checker except `exact` (which as the name implies does a byte-by-byte equality check).

**Comment by FherStk** [comment]
Sorry, I don't want to be annoying, but even with the "binary_data: true" field, the judge is giving to me the same output:

![image](https://user-images.githubusercontent.com/324326/207077860-f1059da4-d016-4ede-a721-737b965d245a.png)

Any clues?

Thanks again,
Fer

**Comment by kiritofeng** [comment]
Can you try `hints: [unicode]` instead?

**Comment by FherStk** [comment]
Great! With "hints" it works as I expected. So... I'll need to change my pull request to add this too, any clues about how to handle it? Because "hints" is a collection and "binary_data" was just a boolean.

Thanks!
Fher
